src/components/pages/Cases/CaseTable.js

```diff
@@ -24,17 +24,18 @@ import {
 } from 'antd';
 import './CaseTable.less';
 import CaseDetails from '../CaseOverview/CaseDetails';
-//
-// Case column utils import
-import { case_columns } from '../../../utils/case_utils/case_columns';
+//* Case column utils import
```

Comment:
Not sure of the reason why you added an asterisk to the start of the comments. I would delete all of those!

Reply:
It is actually an extension called better comments, it helps pop out instead of it being grey. I deleted it though, I guess it has some people at a loss if they are not used to using better comments.